memory.tex

```diff
@@ -1732,8 +1734,8 @@ \subsection{C64-style Memory Mapping}
 addresses 0.0000 and 0.0001 that are normally hidden by the C64-style banking
 registers. If the MAP register selects the \$0000 -- \$1FFF address block for
 mapping and applies an offset of 0, 16-bit addresses \$0000 and \$0001 will
-access RAM at 0.0000 and 0.0001, and will not access the C64-style memory
-mapping registers. The C64-style banking registers can only be accessed with
+access RAM at 0.0000 and 0.0001, and will not access the C64-style banking
+registers. The C64-style banking registers can only be accessed with
```

Comment:
The term "C64-style banking registers" is repeated in those sentences. Maybe consider saying "They" or "The latter" in the second sentence?

Reply:
Acknowledged, it's a mouthful. In this case the paragraph is talking about both MAP and 00/01 so I want to be extra clear.